lib/bump/cli/commands/base.rb

```diff
@@ -24,15 +24,18 @@ def body(file, **options)
           compact(
             {
               definition: definition.content,
+              references: prepare_references(definition.external_references),
               specification: options[:specification],
               validation: options[:validation],
               auto_create_documentation: options[:'auto-create']
             }.merge(documentation_or_hub(options))
           )
         end
 
-        def prepare_file(file, options)
```

Comment:
I think this removal should belong to first commit (4da6e48), cf related comment

Reply:
:+1: